app/src/ui/changes/commit-message-avatar.tsx

```diff
@@ -146,6 +148,15 @@ export class CommitMessageAvatar extends React.Component<
             ))}
           </Select>
         </Row>
+        <Row>
+          <div className="secondary-text">
+            You can also choose an email only for this repository from the{' '}
```

Comment:
```suggestion
            You can also choose an email specific to this repository from the{' '}
```

Reply:
Don't know why but "only for" feels funny to me.. but could just be me 🙂  - I don't have any strong feelings on this so feel free to ignore the suggestion.